docs_user/modules/proc_stopping-openstack-services.adoc

```diff
@@ -137,15 +137,22 @@ PacemakerResourcesToStop=("openstack-cinder-volume"
 
 echo "Stopping systemd OpenStack services"
 for service in ${ServicesToStop[*]}; do
+ifeval::["{build_variant}" != "ospdo"]
     for i in {1..3}; do
-        SSH_CMD=CONTROLLER${i}_SSH
-        if [ ! -z "${!SSH_CMD}" ]; then
-            echo "Stopping the $service in controller $i"
-            if ${!SSH_CMD} sudo systemctl is-active $service; then
-                ${!SSH_CMD} sudo systemctl stop $service
-            fi
+    SSH_CMD=CONTROLLER${i}_SSH
+endif::[]
+ifeval::["{build_variant}" == "ospdo"]
+    SSH_CMD=CONTROLLER_SSH
+endif::[]
+    if [ ! -z "${!SSH_CMD}" ]; then
+        echo "Stopping the $service in controller $i"
+        if ${!SSH_CMD} sudo systemctl is-active $service; then
+            ${!SSH_CMD} sudo systemctl stop $service
```

Comment:
This still un-indents things for the default guide. Let's please keep the original indentation, it will also make the diff cleaner like in the other places where we do similar edits.

Reply:
ok update that to indent correctly

Reply:
I don't think it's fixed... If it was, the lines of diff would disappear. The `if` line is indented by 4 spaces but it was previously indented by 8 spaces and i think we should keep that, to make it appear correct in the `for` cycle. 

See how you did it in `proc_migrating-ovn-data.adoc`, i think there it is correct.

Reply:
If i look at the docs preview for this PR, this is how it renders:

```
echo "Stopping systemd OpenStack services"
for service in ${ServicesToStop[*]}; do
    for i in {1..3}; do
        SSH_CMD=CONTROLLER${i}_SSH
    if [ ! -z "${!SSH_CMD}" ]; then
        echo "Stopping the $service in controller $i"
        if ${!SSH_CMD} sudo systemctl is-active $service; then
            ${!SSH_CMD} sudo systemctl stop $service
        fi
    fi
    done
done
```

Reply:
updated indentation , I believe its better now , now it's looking like this : 
![image](https://github.com/user-attachments/assets/b58b8b5b-2e2d-4811-bf52-be7d6a73ac9d)